src/database/migrations/0019-create-organizations_proposals.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE organizations_proposals (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  organization_id INTEGER NOT NULL,
+  proposal_id INTEGER NOT NULL,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  FOREIGN KEY (organization_id) REFERENCES organizations(id),
+  FOREIGN KEY (proposal_id) REFERENCES proposals(id),
+  UNIQUE(organization_id, proposal_id)
```

Comment:
Scratch the comment I just left (which I deleted). _Of course_ this uniqueness constraint doesn't mean we can't have 1:many. (It does mean we can't have `many:many` but we knew that.)

Reply:
It means you cannot have more than one TYPE of connection between an org and a proposal, but an org can have many proposals, and a proposal can have many orgs (which is what I think of when I say `many:many`)